### PR TITLE
windows popup

### PR DESCRIPTION
**Pull request by alizaamir123**
##In this PR:
When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
When the user clicks (or taps) the close (X) button, the popup disappears.
You need to use a JavaScript array to store all of the information for all projects.

### CODE REVIEW

**Review by Zuhaib042 — COMMENTED** [review]
Hi @alizaamir123 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is a pair programming activity and I don't see any commit from your partner. kindly consider getting commit from your partner as actual code not just some changes in readme.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: Js/popup.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+         <li class="name">${data.frame[0]}</li>
+         <li class="skills"><img src="${data.dot}" alt="">
+         <p>${data.frame[1]}</p></li> 
+         <li class="year"><img src="${data.dot}" alt="">
+           <p>${data.frame[2]}</p></li>
+     </ul>
+   </div>
+   </div>
+     <div class="image">
+       <img class="img pop-image" src="${data.popimage}" alt="">
```
Comment (vickymarz) [inline Js/popup.js:24]: - [ ] Kindly fix the image overstretch on all your popup cards for the desktop view. You can fix this by removing the fixed height you added for the images.
your image 👇🏽 
image
 
figma image 👇🏽 
image

File: Js/popup.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
+import popupdata from './data.js';
+
+const submit = document.querySelectorAll('.see-project');
+const popopcontainer = document.querySelector('.popup-container');
+const popupwindow = (data) => {
+  popopcontainer.innerHTML = ` <div class="container-popup">
```
Comment (vickymarz) [inline Js/popup.js:6]: - [ ] Some content on your popup seems to be chopped off on both mobile and desktop views. I think you can fix this by removing the margin-bottom of the container ad setting the popup min-height to 100%

image

**Review by vickymarz — CHANGES_REQUESTED** [review]
Hi @alizaamir123, 

You have done a great job so far! 💪 👏
You still need to work on a few issues before moving on to the next project, but you're almost there!

## Top highlight 🎯

- The Pull Request is following the Git Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The README is well documented ✅ 
- Linters are working properly ✅ 


## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by jmonto55 — APPROVED** [review]
# STATUS: APPROVED 🟢

Hi @alizaamir123 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

AgreeSealGIF


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**